Samples/MapScenarios/VoicePrompt.md

```diff
@@ -0,0 +1,71 @@
+## Sample - Voice Prompt
+====================
+
+### Overview
+- This feature will read route instruction aloud to the user as they swipe through the route instructions or as they traverse them with indoor location.
+
+### Usage
+
+- Need to fill out `applicationId`, `accessKey`, `signatureKey`, and `buildingId` in `strings.xml` and `integers.xml`.
+
+### Sample Code
+- [VoicePromptActivity.kt](https://github.com/phunware/maas-mapping-android-sdk/blob/sample_code_updates/Samples/kotlin/src/main/java/com/phunware/kotlin/sample/VoicePromptActivity.kt)
+- [NavigationOverlayView.kt](https://github.com/phunware/maas-mapping-android-sdk/blob/sample_code_updates/Samples/kotlin/src/main/java/com/phunware/kotlin/sample/NavigationOverlayView.kt)
+- [ManeuverDisplayHelper.kt](https://github.com/phunware/maas-mapping-android-sdk/blob/sample_code_updates/Samples/kotlin/src/main/java/com/phunware/kotlin/sample/ManeuverDisplayHelper.kt)
+
+**Step 1: Copy the following files to your project**
+
+- VoicePromptActivity.kt
+- NavigationOverlayView.kt
+- ManeuverDisplayHelper.kt
+
+Note: For text-to-speech, we use text-to-speech API from `android.speech.tts.TextToSpeech` class
+
+**Step 2: Pay attention to this method `onManeuverChanged`**
+
+```
+override fun onManeuverChanged(navigator: Navigator, position: Int) {
+        // Update the selected floor when the maneuver floor changes
+        val maneuver = navigator.maneuvers[position]
+        val selectedPosition = floorSpinner.selectedItemPosition
+        for (i in 0 until floorSpinnerAdapter.count) {
+            val floor = floorSpinnerAdapter.getItem(i)
+            if (selectedPosition != i && floor != null && floor.id == maneuver.floorId) {
+                floorSpinner.setSelection(i)
+            }
+        }
+
+        //if voice is enabled 
+        if (voiceEnabled) {
+            val pair = navOverlay.getManeuverPair()
+            var text = displayHelper.stringForDirection(this, pair.mainManeuver)
+
+            val turnManeuver  = pair.turnManeuver
+            var turnable : Boolean  = false
+            if(turnManeuver != null)
```

Comment:
Let's do Java/Kotlin style code formatting for code snippets too.

Reply:
will do this in the other PR

Reply:
done